documentation/docs/plugins/deploy.md

```diff
@@ -0,0 +1,40 @@
+---
+displayed_sidebar: pluginsSidebar
+---
+
+# Deploying your Plugin
+
+As previously mentioned, plugins must be hosted on your own infrastructure. This gives you full control over the technologies you use and when to update your plugin(s).
+
+## Where to Deploy
+
+You can deploy your plugin/app anywhere. But it's import to make the plugin accessible publicly from the web. Otherwise, the features of your plugin will not be visible to users.
+
+## CORS Enabled
+
+The `manifest.json` of your application should be served using a CORS-enabled HTTP server. Since everything this file is fetching from our application, this must be enabled on your infrastructure in order to allow us to fetch your `manifest.json` content and enable related features.
```

Comment:
```suggestion
The `manifest.json` of your application should be served using a CORS-enabled HTTP server. Since this file fetches data from our application, this must be enabled on your infrastructure in order to allow us to fetch your `manifest.json` content and enable related features.
```